src/v2/Apps/Artist/Routes/Overview/Components/ArtistRelatedArtistsRail.tsx

```diff
@@ -140,9 +139,8 @@ export const ArtistRelatedArtistsRailFragmentContainer = createFragmentContainer
                     node {
                       internalID
                       slug
-
                       image {
-                        resized(width: 325, height: 230) {
+                        cropped(width: 325, height: 230) {
```

Comment:
👍  -- this actually fixes something I missed in the banner; was using `resized` instead of `cropped`.